docs/reference/modules/snapshots.asciidoc

```diff
@@ -60,6 +60,9 @@ or
 GET /_snapshot/_all
 -----------------------------------
 
+The `ignore_unavailable` parameter can be used to controls whether to ignore if any specified snapshots are unavailable.
+Either true or false can be specified.
+
```

Comment:
I would put something below

```
All snapshots currently stored in the repository can be listed using the following command:

GET /_snapshot/my_backup/_all
```

The command fails if some of the snapshots are unavailable. The boolean parameter `ignore_unvailable` can be used to return all snapshots that are currently available.